embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -85,6 +85,8 @@ internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
             val embAndroidReactNativeCrashJsException = EmbraceAttributeKey("android.react_native_crash.js_exception")
         }
 
+        internal object ReactNativeAction : System("rn_action", true)
```

Comment:
Should all React Native telemetry go into the same name space? So this becomes `react_native.action` or something?